espaloma/data/qcarchive_utils.py

```diff
@@ -222,3 +222,36 @@ def h5_to_dataset(df):
         gs.append(g)
 
     return esp.data.dataset.GraphDataset(gs)
+
+def breakdown_along_time_axis(g, batch_size=32):
```

Comment:
should mention in docstring that (1) it modifies g in-place in addition to returning a copy, (2) it returns randomly sampled snapshots, not in continuous chunks along time axis

Reply:
Oh, shoot, sorry -- I misread this! It only modifies idxs in place not g